### PR TITLE
[camera][document-picker] Fix broken links

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Closes ENG-10085

# How

Fix broken links to FileSystem API doc methods from Expo Camera and Expo Document Picker. Changes applied to unversioned docs.

# Test Plan

Run docs locally and visit: http://localhost:3002/versions/unversioned/sdk/camera/#takepictureasyncoptions & http://localhost:3002/versions/unversioned/sdk/document-picker/#documentpickeroptions

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by lukmccall — APPROVED** [review]

**Review by Simek — APPROVED** [review]
LGTM, please remember to rebuild package files before merging. 🙂

**Comment by amandeepmittal** [comment]
@Simek, yeah I need to check what happened there. Something didn't get merge right 😅

File: packages/expo-camera/build/Camera.d.ts
Comment (amandeepmittal) [inline packages/expo-camera/build/Camera.d.ts:1]: @Simek, the following changes are auto generated, do you think they can cause issue for rebuilding the package files?

I reverted the doc change for Expo Camera and I can see these lines in the file are still happening.

File: packages/expo-camera/build/Camera.d.ts
Comment (amandeepmittal) [inline packages/expo-camera/build/Camera.d.ts:26]: @Simek, the following changes are auto generated, do you think they can cause issue for rebuilding the package files?

I reverted the doc change for Expo Camera and I can see these lines in the file are still happening.

**Pull request closed without merging**
